docs/src/pages/components/lists/lists.md

```diff
@@ -47,7 +47,7 @@ Below is an interactive demo that lets you explore the visual results of the dif
 
 {{"demo": "pages/components/lists/SelectedListItem.js"}}
 
-## Align list items
+## Align List Items
```

Comment:
text shouldn't be used for typography if that makes sense. If we want to title case it should happen via styling. Though we are inconsistent with it:

> Icon Buttons
> Customized buttons

-- https://material-ui.com/components/buttons/

We should apply title case with styling and only use uppercase first for proper nouns. @mbrookes What do you think?

Reply:
+1 for 
```suggestion
## Align list Items
```

Reply:
Why "Items" in uppercase but "list" not?

Reply:
My diff is wrong. I'm in favor of only uppercasing the first letter of the title.

Reply:
However, what do you think of leaving it untouched? I think that it's something we can handle in one pass.

Reply:
I will remove the casing change from this PR. I am happy to create a new PR changing the titles of all components to _Sentence casing_ if that would be helpful. I think having consistent titles casing would be nice.